Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -184,34 +318,56 @@ void HiresTexture::Prefetch()
       }
       if (iter != s_textureCache.end())
       {
-        for (const Level& l : iter->second->m_levels)
+        for (const Level& tex : iter->second->m_levels)
         {
-          size_sum += l.data_size;
+          prefetched_size_sum += tex.data_size;
         }
       }
     }
 
-    if (s_textureCacheAbortLoading.IsSet())
+    if (s_texturePrefetcher_AbortLoading.IsSet())
     {
       return;
     }
 
-    if (size_sum > max_mem)
+    // The important part - Stops prefetching to avoid getting out of RAM completely, crash prone
```

Comment:
I thought the important part of prefetching textures was to prefetch textures...